bridges/BrutBridge.php

```diff
@@ -0,0 +1,128 @@
+<?php
+class BrutBridge extends BridgeAbstract {
+	const NAME = 'Brut Bridge';
+	const URI = 'https://www.brut.media';
+	const DESCRIPTION = 'Returns 5 newest videos by category and edition';
+	const MAINTAINER = 'VerifiedJoseph';
+	const PARAMETERS = array(array(
+			'category' => array(
+				'name' => 'Category',
+				'type' => 'list',
+				'values' => array(
+					'News' => 'news',
+					'International' => 'international',
+					'Economy' => 'economy',
+					'Science and Technology' => 'science-and-technology',
+					'Entertainment' => 'entertainment',
+					'Sports' => 'sport',
+					'Nature' => 'nature',
+				),
+				'defaultValue' => 'news',
+			),
+			'edition' => array(
+				'name' => ' Edition',
+				'type' => 'list',
+					'values' => array(
+						'United States' => 'us',
+						'United Kingdom' => 'uk',
+						'France' => 'fr',
+						'India' => 'in',
+						'Mexico' => 'mx',
+				),
+				'defaultValue' => 'us',
+			)
+		)
+	);
+
+	const CACHE_TIMEOUT = 1800; // 30 mins
+
+	private $feedURI = '';
+
+	private $videoImageRegex = '/https:\/\/img\.brut\.media\/thumbnail\/(?:[a-z0-9-]+)-([a-z0-9]+-[a-z0-9]+-[a-z0-9]+-[a-z0-9]+-[a-z0-9]+)-(square|landscape|portrait)(?:-auto)?\.jpg/';
+
+	public function collectData() {
+
+		$this->feedURI = self::URI . '/' . $this->getInput('edition') . '/' . $this->getInput('category');
```

Comment:
Instead of storing the URL in a private variable, simply build the URL in `getURI` and call the function here. Have a look at how other bridges implement this.

For example Twitter:

https://github.com/RSS-Bridge/rss-bridge/blob/3d231a417f3670d3e93fb63e9af9d62a5d17e87d/bridges/TwitterBridge.php#L126-L145

https://github.com/RSS-Bridge/rss-bridge/blob/3d231a417f3670d3e93fb63e9af9d62a5d17e87d/bridges/TwitterBridge.php#L150

This will also ensure that the correct URL is used for the HTML preview (uses `getURI` for that).